src/test/java/com/wikia/webdriver/testcases/mobile/TrackingOptInModalTests.java

```diff
@@ -144,31 +144,29 @@ public void anonUserInEUGetsModalBackWhenCookieExpires() {
   @Execute(asUser = User.ANONYMOUS, trackingOptIn = true)
   public void anonUserInEUGetsModalBackWhenResetsCookiesViaPrivacyPolicyPage() {
     TrackingOptInPage.setGeoCookie(driver, "EU", "DE");
-    new ArticlePage().open();
 
     Log.info("Geo cookie: ", driver.manage().getCookieNamed("Geo").getValue());
-    Assertion.assertFalse(new TrackingOptInPage().isVisible());
 
     PrivacyPolicyPage privacyPolicy = new PrivacyPolicyPage();
     privacyPolicy.navigateToPrivacyPolicyPage();
-    privacyPolicy.clickResetTrackingButton();
+    Assertion.assertFalse(new TrackingOptInPage().isVisible());
 
+    privacyPolicy.clickResetTrackingButton();
     Assertion.assertTrue(new TrackingOptInPage().isVisible());
   }
 
   @Test(groups = {"mobile-wiki-tracking-opt-in"})
   @Execute(asUser = User.USER, trackingOptIn = true)
-  public void loggedInUserInEUGetsModalBackWhenResetsCookiesViaPrivacyPolicyPage() {
+  public void loggedInUserInEUGetsModalBackWhenResetsCookieFprisViaPrivacyPolicyPage() {
```

Comment:
Typo. Probably by mistake you typed some additional letters inside "loggedInUserInEUGetsModalBackWhenResetsCookie**Fpris**ViaPrivacyPolicyPage"